packages/create-docusaurus/templates/shared/docs/intro.md

```diff
@@ -3,33 +3,54 @@ sidebar_position: 1
 ---
 
 # Tutorial Intro
+A beginners guide <br />
```

Comment:
This is the reason why prettier is unhappy: if you have actually run prettier locally, you would know it should be

```md
# Tutorial Intro

A beginners guide
```

Two consecutive line breaks automatically change paragraphs. No need for `<br />`.